_episodes/07-plot.md

```diff
@@ -129,34 +130,19 @@ src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/gd1-5.pn
-> > steps.  Reading right-to-left, top-to-bottom, we see selection 
-> > based on proper motion, the results of the first selection, 
-> > selection based on color and magnitude, and the results of the 
-> > second selection.  So this figure documents the methodology and 
-> > presents the primary result.
-> > 
-> > 3. The figure is mostly black and white, with minimal use of color, so 
-> > it will work well in print.  The annotations in the bottom 
-> > left panel guide the reader to the most important results.  
-> > It contains enough technical detail for a professional audience, 
-> > but most of it is also comprehensible to a more general audience.  
-> > The two left panels have the same dimensions and their axes are 
-> > aligned.
-> > 
-> > 4. Since the panels represent a sequence, it might be better to 
-> > arrange them left-to-right.  The placement and size of the axis 
-> > labels could be tweaked.  The entire figure could be a little 
-> > bigger to match the width and proportion of the caption.  
-> > The top left panel has unused white space (but that leaves 
-> > space for the annotations in the bottom left).
+> > No figure is perfect, and everyone can be a critic. Here are some topics that could come up in this discussion:
+> >
+> > 1. The primary result is that adding physical selection criteria makes it possible to separate likely candidates from the background more effectively than in previous work, which makes it possible to see the structure of GD-1 in “unprecedented detail,” allowing the authors to detect that the stream is larger than previously observed.
+> >
+> > 2. The figure documents the selection process as a sequence of reproducible steps, containing enough information for a skeptical reader to understand the authors' choices. Reading right-to-left, top-to-bottom, we see selection based on proper motion, the results of the first selection, selection based on stellar surface properties (color and magnitude), and the results of the second selection. So this figure documents the methodology, presents the primary result, and serves as reference for other parts of the paper (and presumably, talk, if this figure is reused for colloquia).
+> >
+> > 3. The figure is mostly black and white, with minimal use of color, and mostly uses large fonts. It will likely work well in print and only needs a few adjustments to be accessible to low vision readers and none to accomodate those with poor color vision. The annotations in the bottom left panel guide the reader to the results discussed in the text.
+> > The panels that can have the same units, dimensions, and their axes are aligned, do.
+> >
+> > 4. The on-sky positions likely do not need so much white space.
+> > Axes ticks for the on-sky position figures are not necessary since this is not in an intuitive coordinate system or a finder chart. Instead, we'd suggest size bar annotations for each dimension to give the reader the needed scale.
```

Comment:
```suggestion
> > Axes ticks for the on-sky position figures are not necessary since this is not in an intuitive coordinate system or a finder chart. Instead, we would suggest size bar annotations for each dimension to give the reader the needed scale.
```